include/tvm/relax/transform.h

```diff
@@ -481,6 +481,18 @@ TVM_DLL Pass DeadCodeElimination(Array<runtime::String> entry_functions);
  */
 TVM_DLL Pass ToMixedPrecision(const DataType& out_dtype);
 
+/*!
+ * \brief The pass is designed for few shot tuning for static shape PrimFuncs. It examines all the
+ *  blocks within the PrimFunc and conducts loop fusion, splitting, and other transformations based
+ *  on MetaSchedule schedule rules but directly samples from the search space instead of using the
+ *  tuning algorithm. User can specify the number of valid counts to try and whether to use runner
+ *  for evaluation.
+ * \param valid_count The number of valid counts to try.
+ * \param runner The runner to evaluate the generated schedules.
+ * \return The Pass.
+ */
+TVM_DLL Pass FewShotTuning(Integer valid_count, ObjectRef runner);
```

Comment:
Passing `runner` might be too much overhead for most of the cases. 
Can we use the default runner and use the boolean argument like `enable_runner`?

Reply:
Yeah sure, will do!